src/features/PesterTests.ts

```diff
@@ -18,7 +18,7 @@ export class PesterTestsFeature implements vscode.Disposable {
     private invokePesterStubScriptPath: string;
 
     constructor(private sessionManager: SessionManager) {
-        this.invokePesterStubScriptPath = path.resolve(__dirname, "../../../InvokePesterStub.ps1");
+        this.invokePesterStubScriptPath = path.resolve(__dirname, "../../../modules/PowerShellEditorServices/InvokePesterStub.ps1");
```

Comment:
I will need to run the tests internally so I can merge https://github.com/PowerShell/PowerShellEditorServices/pull/1513 first.

Reply:
It bothers me that the tests pass, because they shouldn't without that PR being merged. 😢

Reply:
Works!